app/components/body-layout.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Container, Heading } from '@chakra-ui/react';
+
+interface LayoutProps {
+  heading: string;
+  children: React.ReactNode;
+}
+
+export default function BodyLayout({ heading, children }: LayoutProps) {
+  return (
+    <Container maxW="container.xl" px={{ base: 4, md: 45 }}>
+      <Heading as="h1" size={{ base: 'lg', md: 'xl' }} mt={{ base: 6, md: 12 }}>
+        {heading}
+      </Heading>
+      {children}
+    </Container>
```

Comment:
You could put this `Container` around the `Outlet` in [app/routes/_index.tsx](https://github.com/DevelopingSpace/starchart/blob/7bb2b2652f9b8c490f5cd43f919e85d508a8434c/app/routes/__index.tsx) instead and have it render automatically instead of having to import this component on every index route.
Then, the `Heading` can stay in its own respective page.

Reply:
Do we want to have tables go wider than a `Container` (admin, dns records)?  Or should we constrain that too?

Reply:
Do you mean have the tables go to different widths than the text?
I think we can constrain the text separately as needed, and this will constrain the entire page to have consistent sizes across all pages

Reply:
Right: do we want the tables to be wider than a Container?  Maybe not, since we can put max widths on the columns too and have it all fit without scrolling.

Reply:
Yea, that makes sense. I think the container size  (`xl`) is big enough that we won't need to do anything different for the tables

Reply:
I increased all width on certificate page and replaced the layout template with wrapping root index with Container. Please review if everything is good. Thanks!